### PR TITLE
Add different verbose levels

### LINKED ISSUE #228: Add different verbose levels
Currently we have a `verbose` option  (`-v` / `--verbose` in the CLI) to have `debug` logs enabled.

However, we have tremendous amount of logs (which are really-really useful at different points) which just floods the CLI. We should have multiple levels of verbosity to help this.

### PR DESCRIPTION
**Pull request by kukovecz**
Fixes #228

### CODE REVIEW

**Review by QKaiser — APPROVED** [review]
Looks great. I like the fact that we have a shared function to use in our internal extractors such as unromfs. Levels are well defined, we just have to keep this in mind when writing new handlers :)

File: unblob/finder.py [hunk 1]
```diff
@@ -77,13 +78,15 @@ def search_chunks_by_priority(  # noqa: C901
                         "File format is invalid",
                         exc_info=exc,
                         handler=handler.NAME,
+                        verbosity=2,
```
Comment (vlaci) [inline unblob/finder.py:81]: isn't it `_verbosity`?

File: unblob/finder.py [hunk 2]
```diff
@@ -77,13 +78,15 @@ def search_chunks_by_priority(  # noqa: C901
...
                     )
                     continue
                 except EOFError as exc:
                     logger.debug(
                         "File ends before header could be read",
                         exc_info=exc,
                         handler=handler.NAME,
+                        verbosity=2,
```
Comment (vlaci) [inline unblob/finder.py:89]: isn't it `_verbosity`?

**Comment by vlaci** [comment]
I am thinking about that can't we add our own override methods to the logger via the `wrapper_class` machinery?
Level 1 can remain debug, level 3 could be trace and we'd need to come up with a name for level 2. This way it would be less prone to typeos. Another thing we could do is to override the `debug` method to make the verbosity the first mandatory argument, like `debug(2, "whatever")`.

**Comment by QKaiser** [comment]
> I am thinking about that can't we add our own override methods to the logger via the `wrapper_class` machinery? Level 1 can remain debug, level 3 could be trace and we'd need to come up with a name for level 2. This way it would be less prone to typeos. Another thing we could do is to override the `debug` method to make the verbosity the first mandatory argument, like `debug(2, "whatever")`.

In terms of levels, Apache uses these:
- debug
- trace1
- trace2
- traceX

We would only need to implement trace1 and trace2 for levels 2 and 3 if we go this way.

**Comment by vlaci** [comment]
We were looking it into more deeply with @kukovecz and there is no easy composable way of implementing this on top of structlog. There is one global configuration for logger wrapping, so we cannot adjust the interface just for ourselves.

File: unblob/finder.py (hunk 1 above)
Comment (kukovecz) [reply unblob/finder.py]: Yeah, typo :/

File: unblob/finder.py (hunk 2 above)
Comment (kukovecz) [reply unblob/finder.py]: Yeah, typo :/

**Review by vlaci — APPROVED** [review]

**Pull request merged**